packages/vue/src/components/SpAssets/SpAssets.vue

```diff
@@ -57,26 +76,74 @@ export default defineComponent({
       )
     })
 
+    const filteredBalanceList = computed(() => {
+
+      let searchArray = balances.value,
+        searchString = state.searchQuery
+
+      console.log({searchString})
+
+      if (!searchString) {
+        return sortList(searchArray)
+      }
+
+      searchString = searchString.trim().toLowerCase()
+
+      searchArray = searchArray.filter((item) => {
+        if (item.denom.toLowerCase().indexOf(searchString) !== -1) {
+          return item
+        }
+      })
+
+      // Return an array with the filtered data.
+      return sortList(searchArray)
+    })
+
     // lh
     onMounted(async () => {
       await queryAllBalances({
-        params: { address: props.address },
+        params: { address: props.address},
+        query: {"pagination.reverse": false},
         options: { all: true, subscribe: true }
       })
     })
 
     return {
-      balances
+      filteredBalanceList,
+      ...toRefs(state),
     }
   }
 })
+
+function sortList(list) {
```

Comment:
shouldn't sortList be a computed?

Reply:
This is the main logic that will probably be changed.